sphinx_needs/config.py

```diff
@@ -365,6 +408,54 @@ def get_default(cls, name: str) -> Any:
         default_factory=list, metadata={"rebuild": "env", "types": (list,)}
     )
     """Path to the root table in the toml file to load configuration from."""
+    schemas: SchemasFileRootType = field(
```

Comment:
please rename this to something like `schemas_root` or whatever makes sense,
so that all schema options are `schema_<field>`
ideally I think we should have in the `ubproject.toml` for this to be under its own namespace e.g.

```toml
[needs.schema]
field = ...
```

We could do some magic in the `from_toml` function to support this

Reply:
Done